airbyte-integrations/connectors/source-twilio/acceptance-test-config.yml

```diff
@@ -1,35 +1,53 @@
 connector_image: airbyte/source-twilio:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_twilio/spec.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.10"
+    tests:
+      - spec_path: "source_twilio/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/no_empty_streams_catalog.json"
-      expect_records:
-        path: "integration_tests/expected_records.txt"
-      empty_streams: ["alerts"]
-      timeout_seconds: 600
+    tests:
+      - config_path: "secrets/config.json"
+        expect_records:
+          path: "integration_tests/expected_records.txt"
+        empty_streams:
+          - name: "alerts"
+            bypass_reason: "alerts are transient - they emerge and disappear from time to time"
+          - name: "dependent_phone_numbers"
+            bypass_reason: "stream not filled yet"
+          - name: "conference_participants"
+            bypass_reason: "stream not filled yet"
+          - name: "available_phone_numbers_local"
+            bypass_reason: "very volatile data"
+          - name: "available_phone_numbers_mobile"
+            bypass_reason: "very volatile data"
+          - name: "available_phone_numbers_toll_free"
+            bypass_reason: "very volatile data"
+          - name: "usage_records"
+            bypass_reason: "very volatile data"
```

Comment:
I logged an issue to get better seeding on this account: https://github.com/airbytehq/airbyte/issues/19876